src/main/java/roomescape/reservation/controller/ReservationRestController.java

```diff
@@ -76,11 +80,44 @@ public ResponseEntity<List<AvailableReservationTimeResponse>> getAvailableReserv
 
     @GetMapping("/mine")
     public ResponseEntity<List<ReservationMineResponse>> getMyReservations(final Member member) {
-        final List<Reservation> reservations = reservationService.findByMember(member);
-        final List<ReservationMineResponse> reservationMineResponses = reservations.stream()
+        final List<ReservationMineResponse> reservationMineResponses = reservationService.findByMember(member)
+                .stream()
                 .map(ReservationMineResponse::from)
                 .toList();
+        final List<ReservationMineResponse> waitingMineResponses = reservationService.findWaitingByMember(member)
+                .stream()
+                .map(waiting -> {
+                    final long rank = reservationService.getRankInWaiting(waiting);
+                    return ReservationMineResponse.from(waiting, rank);
+                })
+                .toList();
+
+        final List<ReservationMineResponse> combined = new ArrayList<>();
+        combined.addAll(reservationMineResponses);
+        combined.addAll(waitingMineResponses);
+
+        return ResponseEntity.ok(combined);
+    }
 
-        return ResponseEntity.ok(reservationMineResponses);
+    @PostMapping("/waiting")
```

Comment:
RESTful API에서 자원을 표현할 때는 복수 형태로 표현해주세요 😄 

```suggestion
    @PostMapping("/waitings")
```

참고 : https://meetup.nhncloud.com/posts/92

Reply:
감사합니다!